src/adapter/graphql/src/mutations/flows_mut/dataset_flow_runs_mut.rs

```diff
@@ -125,7 +125,7 @@ enum TriggerFlowResult {
 #[derive(SimpleObject)]
 #[graphql(complex)]
 struct TriggerFlowSuccess {
-    pub flow: Flow,
+    pub flow: Box<Flow>,
```

Comment:
Related to my other comment - a box on this level seems a bit weird. If the `Flow` is getting too big for the stack or is getting too expensive to move, then `the Flow` itself should probably box its state internally. It's unclear to me why we decide to box on the GQL level, but not elsewhere.

Reply:
Note, it's not the `Flow` from the FS, it's the API `Flow` equivalent, which holds FS's `FlowState`. I agree, putting a `Box` 1 level below made the code cleaner.